React/my-app/.gitignore

```diff
@@ -15,4 +15,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
+enviorment.js
```

Comment:
You are move all api configuration to environment file!!!
In env, just put the
 baseUrl String
 And any other key if you have more.

So you shoud export this from environment file to the api.js file que axios is defined!

Reply:
Word is environment
environment.js   take care with the typos